src/index.ts

```diff
@@ -244,6 +246,10 @@ async function getBuildOptions(
     plugins.push(ShebangPlugin(cwd))
   }
 
+  if (stub) {
```

Comment:
We should disable some plugins on stub mode, like dts, report, external, unused...